infra/base-images/base-runner/run_fuzzer

```diff
@@ -134,11 +136,11 @@ elif [[ "$FUZZING_ENGINE" = honggfuzz ]]; then
   # -P: use persistent mode of fuzzing (i.e. LLVMFuzzerTestOneInput)
   # -f: location of the initial (and destination) file corpus
   # -n: number of fuzzing threads (and processes)
-  CMD_LINE="$OUT/honggfuzz -n 1 --exit_upon_crash -R /tmp/${FUZZER}_honggfuzz.report -W $FUZZER_OUT -v -z -P -f \"$CORPUS_DIR\" $(get_dictionary) $* -- \"$OUT/$FUZZER\""
+  CMD_LINE="$OUT/honggfuzz -n 1 --exit_upon_crash -R /tmp/${FUZZER}_honggfuzz.report -W $FUZZER_OUT -v -z -P -f \"$CORPUS_DIR\" $(get_dictionary) $FUZZER_EXTRA_ARGS $* -- \"$OUT/$FUZZER\""
 
-else
+else # it's libfuzzer
 
-  CMD_LINE="$OUT/$FUZZER $FUZZER_ARGS $*"
+  CMD_LINE="$OUT/$FUZZER $FUZZER_ARGS $FUZZER_EXTRA_ARGS $*"
```

Comment:
Oh, you know what I realized, We don't need FUZZER_EXTRA_ARGS, we already have FUZZER_ARGS, can you change this script to:
1. Get rid of FUZZER_EXTRA_ARGS
2. Use FUZZER_ARGS with each fuzzer (not just libFuzzer as was the case before).
Sorry for giving you bad instructions with FUZZER_EXTRA_ARGS previously.

Reply:
Fixed this with https://github.com/google/oss-fuzz/pull/5924/commits/e703b5cf66fbe7f704edc73b079a75110d2b1cc7